scripts/griddle.jsx

```diff
@@ -689,11 +695,8 @@ var Griddle = React.createClass({
         var isFound;
 
         if(isChecked) {
-            isFound = find(selectedRowIds, function (item) {
-                return id === item;
-            });
-
-            if(isFound === undefined) {
+            isFound = selectedRowIds.indexOf(id);
```

Comment:
Yes, about 1 sec. But I was using Timeline to compare the difference, I can write code to get a more precise result.